pkg/kn/commands/service_describe.go

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"errors"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+var serviceDescribePrintFlags *genericclioptions.PrintFlags
+
+// NewServiceDescribeCommand represents the describe command
+func NewServiceDescribeCommand(p *KnParams) *cobra.Command {
+
+	serviceDescribePrintFlags := genericclioptions.NewPrintFlags("").WithDefaultOutput("yaml")
```

Comment:
Do we want the `describe` command to output into `yaml` by default ? (ps: it's also fine to merge like that and revise the default output format later on :angel:)

Reply:
@vdemeester I think there's some discussion going on regarding the output format https://github.com/knative/client/issues/14#issuecomment-459614902 so I plan to update this after it's finalized

Reply:
i've added this to our WG agenda last week.